pkg/controller/jobframework/reconciler.go

```diff
@@ -74,10 +74,11 @@ type Options struct {
 	WaitForPodsReady           bool
 	KubeServerVersion          *kubeversion.ServerVersionFetcher
 	// IntegrationOptions key is "$GROUP/$VERSION, Kind=$KIND".
-	IntegrationOptions map[string]any
-	EnabledFrameworks  sets.Set[string]
-	ManagerName        string
-	LabelKeysToCopy    []string
+	IntegrationOptions        map[string]any
+	EnabledFrameworks         sets.Set[string]
+	EnabledExternalFrameworks []string
```

Comment:
```suggestion
	EnabledExternalFrameworks sets.Set[string]
```

Reply:
I'm not sure about the advantage of `sets.Set` here because the `EnabledExternalFrameworks` is used only in https://github.com/kubernetes-sigs/kueue/blob/ddbbfddc7d7c23de06725cff850e3f8c3748dd54/pkg/controller/jobframework/setup.go#L50-L54

Reply:
Conceptually there should be no duplicates (in my opinion).

Reply:
DONE.